scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -12,44 +12,46 @@
 
 {% block body %}
 <nav class="navbar sticky-top navbar-expand-lg navbar-light bg-light">
-  <!-- Brand and toggle get grouped for better mobile display -->
-  <a class="nav-link" href="{{ url_for('cases.case', institute_id=institute._id, case_name=case.display_name) }}" target="_blank">Case {{case.display_name}}</a>
+  <div class="container-fluid">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <a class="nav-link link-primary mx-2 " style="text-decoration: none !important;" href="{{ url_for('cases.case', institute_id=institute._id, case_name=case.display_name) }}" rel="noopener" target="_blank">Case {{case.display_name}}</a>
```

Comment:
We don't really need the opener link so all good to be safe, but `noopener` won't do much here as the source and dest are the same site. 😊